unit_tests/test_config.py

```diff
@@ -16,9 +16,11 @@
 import itertools
 import unittest
 
-from sdcm.sct_config import SCTConfiguration
-from sdcm.utils.version_utils import resolve_latest_repo_symlink
+from unittest.mock import patch
+
+import sdcm
 
+from sdcm.sct_config import SCTConfiguration
```

Comment:
Following imports

```
from sdcm.sct_config import SCTConfiguration
import sdcm
from sdcm.sct_config import SCTConfiguration  # once again, duplicate
```

And usage:

```
patch.object(sdcm.sct_config, 'get_branch_version', return_value='666.development', clear=True)
...
conf = SCTConfiguration()
```

Looks strange. It must have been just one import:

```
from sdcm import sct_config
...
conf = sct_config.SCTConfiguration()
...
patch.object(sct_config, 'get_branch_version', return_value='666.development', clear=True)
```

From python point of view the work will be done the same but it becomes more readable not having overlapping imports, IMHO.

Also, one of python coding guidelines is to place imports according to it's groups the following way:

```
first-party imports (built-ins)

third-party imports (non built-in libraries we install separately)

local imports (our own code we have locally)
```

So, it would result in following imports here:

```
import itertools
import logging
import os
import unittest
 
from sdcm import sct_config
from sdcm.utils.version_utils import resolve_latest_repo_symlink
```

Reply:
fixed